spec/controllers/articles_controller_spec.rb

```diff
@@ -27,20 +27,21 @@
         end
     end
 
-    describe '#update' do
-      describe 'update a record that already exists,'do
+    describe 'GET #update' do
+      describe 'update a record that already exists'do
         context 'when it finds Article' do
```

Comment:
This is not a context, it is an action that is being applied to the object. This is a expectation.

`it 'finds the requested article' do`